sinatra/app.rb

```diff
@@ -0,0 +1,49 @@
+# sinatra setup prototype
+# OSUOSL March 2016
+
+require 'rubygems'
+require 'bundler/setup'
+require 'sinatra'
+require 'json'
+require 'date'
+
+# preprocess the json from server
+# TODO: change data source to redis
+file = File.read('sample.json')
+data = JSON.parse(file)
+
+# default helloworld
+get '/' do
+  "Hello World!"
+end
+
+# returns bulk data
+get '/data' do
+  content_type:json
+  data.to_json
```

Comment:
I agree with @ElijahCaine here. Leaving out the explicit return is considered by many to be the `ruby way`, but it can be confusing and problematic. See: http://stackoverflow.com/a/13635060